include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3422,8 +3422,9 @@ __pattern_set_intersection(_ExecutionPolicy&& __exec, _RandomAccessIterator1 __f
             [](_DifferenceType __n, _DifferenceType __m) { return ::std::min(__n, __m); },
             [](_RandomAccessIterator1 __first1, _RandomAccessIterator1 __last1, _RandomAccessIterator2 __first2,
                _RandomAccessIterator2 __last2, _T* __result, _Compare __comp) {
-                return oneapi::dpl::__utils::__set_intersection_construct(__first1, __last1, __first2, __last2,
-                                                                          __result, __comp);
+                return oneapi::dpl::__utils::__set_intersection_construct(
+                    __first1, __last1, __first2, __last2, __result, __comp,
+                    [](auto&& a, auto&& b) { return std::forward<decltype(a)>(a); }); //copy first to output
```

Comment:
`std::forward` -> `::std::forward` in this return and the one below.

Reply:
thanks, fixed.